group_vars/repository.yml

```diff
@@ -33,7 +33,7 @@ global_properties:
     cluster:
       enabled: "{{ (groups['repository'] | length > 1 and not (cluster_keepoff | bool)) | lower }}"
   share:
-    host: "{{ fqdn_alfresco | default(known_urls[0]) | default(nginx_host) }}"
+    host: "{{ fqdn_alfresco | default(known_urls[0] | urlsplit('hostname')) | default(nginx_host) }}"
```

Comment:
check this

not sure if I should uniform to L30 (by adding know_urls there or removing here)